src/data-table/data-table.tsx

```diff
@@ -0,0 +1,311 @@
+                                : rowsConfig?.className
+                        }
+                        alternating={isAlternatingRow(index)}
+                        isSelectable={selectionConfig?.showCheckboxes}
+                        isSelected={isRowSelected(row.id.toString())}
+                    >
+                        {selectionConfig?.showCheckboxes &&
+                            renderRowCheckBox(row.id.toString())}
+
+                        {headers.map((header) => renderRowCell(header, row))}
+
+                        {/* {actionsConfig?.showActions &&
+                            actionsConfig.actions && (
+                                <BodyCell
+                                    data-testid={getDataTestId(
+                                        `row-${row.id.toString()}-action`
+                                    )}
+                                    width={actionsConfig.headerWidth}
+                                    className={actionsConfig.className}
+                                >
+                                    {actionsConfig?.actions(
+                                        row,
+                                        isRowSelected(row.id.toString())
+                                    )}
+                                </BodyCell>
+                            )} */}
+                    </BodyRow>
+                ))}
+            </>
+        );
+    };
+
+    const renderRowCell = (header: HeaderProps, row: RowProps) => {
+        const style = typeof header !== "string" ? header.style : undefined;
+        const fieldKey = typeof header === "string" ? header : header.fieldKey;
+        const cellData = row[fieldKey];
+        const cellId = `${row.id.toString()}-${fieldKey}`;
+
+        return (
+            <BodyCell
+                data-testid={getDataTestId(`row-${cellId}`)}
+                key={cellId}
+                style={style}
+            >
+                {cellData}
+            </BodyCell>
+        );
+    };
+
+    const renderRowCheckBox = (rowId: string) => {
+        return (
+            <BodyCell
+                data-testid={getDataTestId(`row-${rowId}-selection`)}
+                className={selectionConfig?.className}
+                width={selectionConfig?.headerWidth}
+            >
+                <CheckBoxWrapper>
+                    <CheckBox
+                        displaySize="small"
+                        checked={isRowSelected(rowId)}
+                        onClick={() => {
+                            selectionConfig?.onClickSelect?.(
+                                "id",
+                                rowId,
+                                !isRowSelected(rowId)
+                            );
+                        }}
+                    />
+                </CheckBoxWrapper>
+            </BodyCell>
+        );
+    };
+
+    const basicEmptyView = () => {
+        return (
+            <ErrorDisplay
```

Comment:
is there a default title/description?

also note that the ErrorDisplay currently doesn't allow empty title/description (it will revert to the placeholder copy). if this functionality is needed, please make the change in a separate PR

separately, the header in ErrorDisplay is H1 but Figma shows H3 for the table. can check with UX if this is ok, otherwise you can override the text styling